pom.xml

```
@@ -12,9 +12,10 @@
   <packaging>pom</packaging>
   <name>Quarkus - Openapi Generator - Parent</name>
   <modules>
-    <module>deployment</module>
-    <module>runtime</module>
-    <module>test-utils</module>
+    <module>client/deployment</module>
+    <module>client/runtime</module>
+    <module>client/test-utils</module>
+    <module>server</module>
```

```suggestion
    <module>client</module>
    <module>server</module>
```

Then we create a pom in `client/pom.xml`. 👍